config/webpack.config.prod.js

```diff
@@ -35,9 +36,7 @@ module.exports = {
     path: buildPath,
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].chunk.js',
-    // TODO: this wouldn't work for e.g. GH Pages.
-    // Good news: we can infer it from package.json :-)
-    publicPath: '/'
+    publicPath: publicPath
```

Comment:
Ah right. Thanks for bearing with me. Added that, so we should now always have the path portion of the url defined in `homepage`, or `/` if it isn't defined.